ui/src/node_modules/app/config.js

```diff
@@ -2,6 +2,7 @@ import regionSpecs from '@svizzle/atlas/src/specs';
 
 export const toolName = 'BEIS/Nesta R&D dashboard';
 export const toolLongName = 'BEIS/Nesta Research & Development spatial data tool';
+export const contactEmail = 'mailto:dataanalytics@nesta.org.uk'
```

Comment:
I think we should remove `mailto:`

```suggestion
export const contactEmail = 'dataanalytics@nesta.org.uk'
```